docs/en/observability/ingest-traces.asciidoc

```diff
@@ -1,10 +1,14 @@
-[[ingest-traces]]
-= Ingest application traces with {agent}
+[[traces-apm-get-started]]
+= Get started with application traces and APM
 
 ++++
-<titleabbrev>Ingest traces</titleabbrev>
+<titleabbrev>Traces and APM</titleabbrev>
 ++++
 
+//TODO: Ask Brandon--can I rename this file? I guess we will need to coordinate
+//with a corresponding update for APM. I just think it's good to rip of the
+//bandaid and give these files accurate names. WDYT?
```

Comment:
That's correct. We'll need a corresponding update and backport here:

https://github.com/elastic/apm-server/blob/087e516ac23277aa2f76bf9749ff367347073cbf/docs/apm-quick-start.asciidoc?plain=1#L9

Reply:
Is this something we can coordinate separately after this PR is merged? That might make things easier.

Reply:
Yah after the PR is merged works.